README.md

```diff
@@ -288,20 +288,22 @@ will match the trim box of the actual artwork for the just jacket.
    "assumedPrintType": 1,
    "totalCirculation": 400,
    "jobType": 0,
+   "orderType": 0,
    "remark": "£15 bundle in 500's cost added to supplier cost total as discussed",
    "machineName": "B3D-Z"
  }
 ```
 
 | Property | Description | Values |
 |---|---|---|
-|additionalProjectName||Additional Order number. Reference number. (String)||
-|completionType|Urgency of the job. Default value is 4 (Standard)|INVALID(0), <br>SAME_DAY(1), <br>EXPRESS(2), <br>OVERNIGHT(3), <br>STANDARD(4), <br>SAVER(5);|
-|totalCirculation|The total number of copies (Integer)||
-|assumedPrintType|The expect print technology|NONE(0), <br>LITHO(1), <br>DIGITAL(2), <br>INKJET(3)|
-|jobType|Classification of print job|PRIMARY_PRINT(0), <br>RIPPED_PROOF(1), <br>HARD_COPY_PROOF(2)|
-|remark|Remarks for the supplier added by Mixam at fulfilment-time, if any (String)||
-|machineName|The name of the recommended machine for this job identified by our pricing engine (String)||
+|additionalProjectName| | Additional Order number. Reference number. (String)                                         ||
+|completionType| Urgency of the job. Default value is 4 (Standard) | INVALID(0), <br>SAME_DAY(1), <br>EXPRESS(2), <br>OVERNIGHT(3), <br>STANDARD(4), <br>SAVER(5); |
+|totalCirculation| The total number of copies (Integer) | |
+|assumedPrintType| The expected print technology | NONE(0), <br>LITHO(1), <br>DIGITAL(2), <br>INKJET(3) |
+|jobType| Classification of print job | PRIMARY_PRINT(0), <br>RIPPED_PROOF(1), <br>HARD_COPY_PROOF(2) |
+|orderType| Classification of order, if applicable | STANDARD(0), <br>FOURTHWALL(1) |
```

Comment:
Thats a better name for this

Reply:
It's too general... but it's better in other ways.